docs/source/dev_notes_database.rst

```diff
@@ -0,0 +1,54 @@
+Database structure
+==================
+
+The database schemas
+--------------------
+
+There are two primary database schemas which the majority of users will works with:
+
+- The "default" schema, which the a hard-coded variable ``SCHEMA_VERSION`` in
+  the ``src/dataregistry/db_basic.py`` file. It can be imported by ``from
+  dataregistry.db_basic import SCHEMA_VERSION``
+- The production schema. This is where production datasets go, and has only
+  read access for the general user. By default this schema is named
+  "production", however during schema creation (see below) you can specify the
+  name of the production schema (though this should only be changed for testing
+  purposes).
+
+Users can specify their own schemas during the initialization of the
+``DataRegistry`` object (by default ``SCHEMA_VERSION`` is connected to). If
+they wish to connect to the production schema its name will have to be manually
+entered (see production schema tutorial). If the user wishes to connect to a
+custom schema they will have to manually enter its name, however they will have
+to have created their own schema for it to work.
+
+When using *SQLite* as the backend (useful for testing), the concepts of
+schemas do no exist.
+
+First time creation of database schemas
+---------------------------------------
+
+In the top level ``scripts`` directory there is a ``create_registry_schema.py``
+script to do the initial schema creation. Before using the data registry, both
+for *Postgres* and *SQLite* backends, this script must be run.
```

Comment:
Let's change "this script must be run" to "this script must have been run".  That is, it isn't necessary that the person using the data registry with that particular schema also be the one who ran the script.  And it only needs to be run once.  With the current language that's not clear.